src/Views/Homepage.vala

```diff
@@ -21,6 +21,27 @@
 public class AppCenter.Homepage : AbstractView {
     private const int MAX_PACKAGES_IN_BANNER = 6;
     private const int MAX_PACKAGES_IN_CAROUSEL = 9;
+    private const string[] pop_picks = {
+        "com.slack.Slack",
+        "org.telegram",
+        "org.gnome.meld",
+        "com.valvesoftware.Steam",
+        "net.lutris.Lutris",
+        "com.mattermost.Desktop",
+        "com.visualstudio.code",
+        "org.gnome.DejaDup",
+        "com.spotify.Client",
+        "com.gexperts.Tilix",
+        "alacritty",
+        "com.uploadedlobster.peek",
+        "virt-manager",
+        "org.signal.Signal",
+        "flameshot",
```

Comment:
Flameshot "flameshot" defaults to the Flatpak and so it never shows up in the list. It need to be changed to "org.flameshot.Flameshot".

Reply:
This should probably be in a separate issue/PR because the current version of Shop (and indeed, every version going back a while) has the same issue.

Reply:
@n3m0-22 if Ian doesn't want to, go ahead and make the pr then.
Also. when you know how something can be changed, you can do [github suggestion](https://stackoverflow.com/questions/54239733/how-can-i-manually-add-suggestions-in-code-reviews-on-github)